db/migrate/20241108040153_cree_questionnaire_litteratie.rb

```diff
@@ -0,0 +1,10 @@
+class CreeQuestionnaireLitteratie < ActiveRecord::Migration[7.0]
+  class ::Questionnaire < ApplicationRecord; end
+
+  def change
+    litteratie = Questionnaire.find_or_create_by(nom_technique: 'litteratie_2024', libelle: "Littératie 2024")
+    question = Question.find_or_create_by(nom_technique: 'sous_consigne_LOdi_1', libelle: "sous consigne LOdi1", type: 'QuestionSousConsigne')
```

Comment:
```suggestion
    question = Question.find_or_create_by(nom_technique: 'sous_consigne_LOdi_1', libelle: "sous consigne LOdi1", type: 'QuestionSousConsigne')
```
Je pense qu'il faut mieux écrire : 
```suggestion
    question = Question.find_or_create_by(nom_technique: 'sous_consigne_LOdi_1') do |q|
    q.libelle = "sous consigne LOdi1"
    q.type = 'QuestionSousConsigne'
    end
```
Sinon on risque de re-créer une nouvelle question alors qu'elle existe déjà mais avec un libellé légèrement différent.